app/lib/Service/ItemService.php

```diff
@@ -903,29 +903,11 @@ public function addItem($ps_table=null, $pa_data=null) {
 					}
 				}
 			}
+		}
```

Comment:
This was previously inside the 'related' processing, and representations don't require related records so I've moved it just outside that `if`